keras_cv/models/generative/stable_diffusion/clip_tokenizer.py

```diff
@@ -109,6 +128,22 @@ def end_of_text(self):
     def start_of_text(self):
         return self.encoder["<|startoftext|>"]
 
+    def add_tokens(self, tokens):
```

Comment:
Maybe this method is deserving of a bit of documentation?

Reply:
(In particular calling out that this is expensive and causes re-creation of the encoder/decoder)